common/djangoapps/third_party_auth/tests/specs/base.py

```diff
@@ -862,10 +867,11 @@ def test_full_pipeline_succeeds_registering_new_account(self):
         # Set the cookie and try again
         self.set_logged_in_cookies(request)
         self.assert_redirect_to_dashboard_looks_correct(
-            actions.do_complete(strategy.request.backend, social_views._do_login, user=created_user))
+            actions.do_complete(strategy.request.backend, social_views._do_login, user=created_user)
+        )
```

Comment:
Yep, I agree, we need clean diffs. But it's in the same file as the others where we're going for that full hanging style -- let's make a separate commit on this that'll be separate from the squash? Or just leave the entire file alone altogether.